source/MRMesh/MRCylinder3.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include "MRLine.h"
+
+namespace MR
+{
+// A class describing a cylinder as a mathematical object.A cylinder is represented by a centerline, a radius, and a length.template <typename T>
+// TODO: Cylinder3 could be infinite. For example for infinite Cylinder3 we could use negative length or length = -1
+template <typename T>
+class Cylinder3
+{
+public:
+    Cylinder3()
+    {}
+
+    Cylinder3( const Vector3<T>& inCenter, const Vector3<T>& inDirectoin, T inRadius, T inHeight )
```

Comment:
May be replace "inHeight" with "inLength"?

Reply:
done